src/System.Private.ServiceModel/tools/scripts/PingWCFService.ps1

```diff
@@ -7,7 +7,7 @@ For ($i=0; $i -le 60; $i++)
 {
     try
     {
-        $request.DownloadFile($Url, "temp.crl")
+        $request.DownloadString($Url)
```

Comment:
What's the impact of this particular change? Does this give us the correct output filename?